README.md

```diff
@@ -91,3 +91,28 @@ httpAuth:
       username: user
       password: pass
 ```
+## Observability
+
+### Logs
+
+There is a middleware using the [zap](https://github.com/uber-go/zap) package to log requests with the JSON zap encoder.
+
+All other pieces of code use the [op](https://github.com/op/go-logging) package with a custom format.
+
+You can see logs in the company's [logz.io](https://app.logz.io/) account, which is accessible via the [Okta user home page](https://wildlifestudios.okta.com/app/UserHome).
+
+⚠️ Only logs formatted as JSON are exported to the logging platform currently in use by the company. That is, only the aforementioned middleware is logging correctly and the team should fix this issue.
+
+### Traces
+
+OpenTracing is used to report traces. The client implementation used is Jaeger's.
+
+You can see traces [here](https://jaeger.wildlifestudios.com/).
+
+⚠️ OpenTracing is deprecated, as well as Jaeger's client package. That is, the team should replace OpenTracing with OpenTelemetry and Jaeger's client package with OpenTelemetry's agnostic client package.
+
+## Metrics
+
+Metrics are exported using [Datadog's statsd](https://github.com/DataDog/datadog-go) client package.
+
+You can see metrics in the company's [Datadog](https://app.datadoghq.com/) account, which is accessible via the [Okta user home page](https://wildlifestudios.okta.com/app/UserHome).
```

Comment:
What do you think of removing this part once it's an open-source project?

Reply:
Right! Thanks for the heads up, I've totally forgot  😅

Can you take a look again?